deploy/helm/sumologic/values.yaml

```diff
@@ -122,6 +122,16 @@ sumologic:
     ## uncomment for the debug mode (disables the automatic run of the setup.sh script)
     # debug: true
 
+    monitors:
+      ## If enabled, a pre-install hook will create k8s monitors in Sumo Logic
+      setupEnabled: true
+
+      ## If true, the installed monitors will be disabled by default
+      monitoringDisabled: false
```

Comment:
I would change this flag to `default` `enableByDefault` or sth like that. There is no need to repeat `monitoring` keyword here